scheduler/scheduler.go

```diff
@@ -149,17 +157,28 @@ func GetDistroQueueInfo(distroID string, tasks []task.Task, maxDurationThreshold
 		}
 
 		duration := task.FetchExpectedDuration().Average
-		task.ExpectedDuration = duration
-		distroExpectedDuration += duration
+
+		if !opts.IncludesDependencies {
+			task.ExpectedDuration = duration
+			distroExpectedDuration += duration
+		} else if depsMet, _ := task.DependenciesMet(depCache); depsMet {
```

Comment:
Make this an `||` so that you don't have to duplicate these lines?

Reply:
yeah.... the assignment in the if statement doesn't seem to compile if its in an || statement.